src/comm_droplet.F

```diff
@@ -686,35 +853,84 @@ subroutine setupDepartDroplet(Depart_ind,ptrDepart,pdata_neighbor)
       sw_idx = ptrDepart(isw)
       se_idx = ptrDepart(ise)
       do np=1, nparcelsLocalActive
-        if(pdata_neighbor(np) .eq. mynorth) then
+        if(pdata_neighbor(np) .eq. inorth) then
```

Comment:
what is the difference between `mynorth` and `inorth`?

Reply:
The variable "mynorth" is the MPI rank of the northern neighbor, while 'inorth' is the index of the northern neighbor.

Reply:
I am still unclear about why this change is necessary. Can you provide an example?

Reply:
Probably best to discuss over a video call.  I made this change based on my understanding of something you told me a number of months ago about a very subtle bug.